lib/components/DateRange/DateRange.js

```diff
@@ -82,6 +85,7 @@ const DateRange = ({
         selectedValues={retrieveDatesFromDateRangeFilterString(activeFilters[name]?.[0])}
         onChange={handleDateRangeChange}
         makeFilterString={makeFilterString}
+        requiredFields={[DATE_TYPES.START]}
```

Comment:
Let's memorize this, since the `requiredFields` is used in `useEffect` deps, it will trigger state in `setFilterValue` all the time.